packages/@react-aria/listbox/docs/useListBox.mdx

```diff
@@ -235,8 +236,10 @@ Use `defaultSelectedKeys` to provide a default set of selected items (uncontroll
 See the `react-stately` [Selection docs](../react-stately/selection.html) for more details.
 
 ```tsx example
+import type {Selection} from 'react-stately';
+
 function Example() {
-  let [selected, setSelected] = React.useState(new Set(['cheese']));
+  let [selected, setSelected] = React.useState<Selection>(new Set(['cheese']));
```

Comment:
These are annoying, and caused by `onSelectionChange` which may emit `"all" | Set<Key>`, but the type of this variable is only inferred to be `Set<string>` so it always requires an annotation.